src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -204,3 +207,59 @@ void ApplicationViewModel::SetMenuCategories()
     // property setter logic.
     Categories = NavCategoryGroup::CreateMenuOptions();
 }
+
+void ApplicationViewModel::AlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e, float width, float height)
+{
+    HandleAlwaysOnTopButtonClick(sender, e, width, height);
+}
+
+task<void> ApplicationViewModel::HandleAlwaysOnTopButtonClick(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e, float width, float height)
+{
+    if (ApplicationView::GetForCurrentView()->ViewMode == ApplicationViewMode::CompactOverlay)
+    {
+        ApplicationDataContainer ^ localSettings = ApplicationData::Current->LocalSettings;
+        localSettings->Values->Insert(WidthLocalSettings, width);
```

Comment:
Unsure of the desired behavior here: if there are multiple calculator windows open, and you move them between AoT and normal modes, the windows will always each "remember" their normal window size.

However, the AoT size is being recorded and used as an app-wide setting, so when a window is transitioned to AoT mode, it will pick up whatever size was last recorded when a window exited AoT.

To see what I mean:

1. Launch two Calc windows.
2. Put window A into AoT and resize to the smallest size.
3. Put window B into AoT. Observe it still goes to the default size (okay).
4. Take window B out of Aot. It goes back to its original size (okay).
5. Take window A out of AoT, then put it back into AoT. It goes back to small size (okay)
6. Put window B into AoT. It goes to the small size, as well, since that was the size last "exited" from (wrong?)

Reply:
FWIW, this is the behavior of Calculator in general (i.e., if you have multiple windows open, we'll restore the last closed window size next time you reopen the app). I agree that there is some behavior to figure out here worth a discussion, but I'd like to see if this becomes a real problem before complicating this logic anymore. @greedyAI, can you open up an issue after this PR is merged in to track that follow up discussion?